flask_logger/tests/test_extension.py

```diff
@@ -1,16 +1,19 @@
 """Test the logger extension module."""
-# pylint: disable=protected-access,redefined-outer-name,unused-variable
+import importlib
+# pylint: disable=protected-access,redefined-outer-name,unused-variable,invalid-name
 import logging
 import unittest
 from unittest.mock import MagicMock, patch
 
 from flask import Flask
 
 from flask_logger import Logger  # isort:skip
-from flask_logger.extension import LOGGERS  # isort:skip
+from flask_logger import extension  # isort:skip
 
 TEST_DSN = 'http://foo:bar@sentry.local/1?timeout=1'  # Got this from raven-python repo tests
 
+extension = importlib.reload(extension)
```

Comment:
Hmm, why is this necessary?

Reply:
It actually isn't I don't think. I was getting test failures in this file saying that Client wasn't defined because I was reloading the module in the other test file. Both test modules get loaded in at the beginning before any execution, so one was breaking the other. I added this line to fix it, but I believe I actually fixed it by changing the import from `from flask_logger.extension import LOGGERS` to `from flask_logger import extension`. I believe that alone fixes it by actually having a reference to the module so it doesn't get refreshed when the other test file reloads it. All of that long-windedness to say, I think you're right - I'll remove this line.

Reply:
Done.

Reply:
👍